.github/workflows/build-test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x]
```

Comment:
We should add `18.x` to this array as well.

Reply:
Requesting permission to wait to add 18.x support to this PR (and the few others) until after vacation.

Reply:
Done